run_python_examples.sh

```diff
@@ -99,6 +99,11 @@ function mnist_hogwild() {
   python main.py --epochs 1 --dry-run $CUDA_FLAG || error "mnist hogwild failed"
 }
 
+function mnist_rnn() {
+  start
+  python main.py --epochs 1 --dry-run || error "mnist rnn example failed"
```

Comment:
noob Q, how does this know to run `mnist_rnn.py`

Reply:
It's a bit hacky. It will cd to `mnist_rnn` directory in `start()` https://github.com/pytorch/examples/pull/1086/files#diff-2f30d316a68c1ff96d3f58961f2dde5fefd5a241687db81e37358a62c2d5d792R52-R53